examples/logic.ml

```diff
@@ -0,0 +1,35 @@
+open Libnacc.Parsing
+open Libnacc.Parsers
+
+type expr =
+  | Atom of char
+  | Lconj of expr * expr
+  | Ldisj of expr * expr
+  | Limpl of expr * expr
+[@@deriving variants, show]
+
+let op_impl = spaced (char '-' *> char '>') *> pure limpl
```

Comment:
Not sure why you would use an explicit sequence of char parsers instead of a literal, it makes more semantic sense I think 🤷🏻

Reply:
juste because we do not have a "literal" parser yet. We should add one (and name it "token" which is a bit more expressive than "literal"). But your absolutely right.

Do you want to add it ? Or we wait for the next patch ?